app/screens/Monobank/Monobank.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react"
+import { View, Text } from "react-native"
+
+import Header from "@components/Header/Header"
+import Input from "@components/Inputs/Input"
+import Button from "@components/Buttons/Button"
+import Link from '@components/Link/Link'
+
+import styles from "./style"
+
+
+const Monobank = ({ navigation }) => {
+    const [activationCode, setActivationCode] = useState(null)
+       
+    const isValid = activationCode 
```

Comment:
For now, it is useless, you can just use for validation activationCode value:  disabled={!activationCode}

Reply:
removed it